src/main/java/com/limechain/sync/warpsync/WarpSyncMachine.java

```diff
@@ -64,7 +64,7 @@ public void start() {
         final Hash256 initStateHash;
         if (this.chainService.getGenesis().getLightSyncState() != null) {
             this.syncedState.setRepository(repository);
-            boolean stateLoaded = this.syncedState.loadState();
+            boolean stateLoaded = !chainService.isLocalChain() && this.syncedState.loadState();
```

Comment:
Not sure if this is expected behaviour. In most cases this would be the intuitive solution but it would confuse some users that want to restart their Host and continue with the sync.

I'd remove `!chainService.isLocalChain()`. If users want to reset their DB, they can use the `-dbc` flag